src/refactor_nrepl/artifacts.clj

```diff
@@ -125,7 +133,8 @@
 (defn artifact-versions
   [{:keys [artifact]}]
   (->> (or (get @artifacts artifact)
-           (get-mvn-versions! artifact))
+           (seq (get-mvn-versions! artifact))
```

Comment:
Might be a good idea to explain the behaviour of this function in the docstring.

Reply:
👍